src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -2001,7 +2001,8 @@ void ExportMusicXml::barlineRight(const Measure* const m, const track_idx_t stra
     const Measure* mmRLst = mmR1->isMMRest() ? mmR1->mmRestLast() : 0;   // last measure of replaced sequence of empty measures
     // note: use barlinetype as found in multi measure rest for last measure of replaced sequence
     BarLineType bst = m == mmRLst ? mmR1->endBarLineType() : m->endBarLineType();
-    bool visible = m->endBarLineVisible();
+    const bool visible = m->endBarLineVisible();
+    const QString color = color2xml(m->endBarLine());
```

Comment:
`m->endBarLine()` can return `0`, and with that `color2xml()` crashes. Probably best to fix it there rather than here.